src/js/index.js

```diff
@@ -0,0 +1,50 @@
+import TaskManager from './logic/taskManager';
+import Task from './models/task';
+import '../css/style.css';
+
+let taskManager = null;
+
+function printInitialTasks() {
+  let savedTasked = JSON.parse(localStorage.getItem('taskDbKey'));
+  if (savedTasked == null) {
+    savedTasked = [];
+  }
+  const taskList = document.getElementById('taskList');
+  taskList.innerHTML = '';
+  taskManager = new TaskManager(savedTasked);
+  taskManager.tasksArray.forEach((task) => {
+    const listViewItem = document.createElement('li');
+    listViewItem.className = 'list-group-item d-flex justify-content-between align-items-center';
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.name = 'name';
+
+    checkbox.className = 'form-check-input pull-left';
+    checkbox.style.marginRight = '17px';
+    checkbox.value = task.completed;
+
+    const span = document.createElement('span');
+    span.className = 'fas fa-ellipsis-v pull-right';
+    listViewItem.appendChild(checkbox);
+    listViewItem.appendChild(document.createTextNode(task.description));
+    listViewItem.appendChild(span);
+    taskList.appendChild(listViewItem);
+  });
+}
+printInitialTasks();
```

Comment:
- [ ] <b>REQUIRED:</b> The project require that there must be at least 3 default projects list on the page. Kindly fix that.